src/main/webapp/movement.js

```diff
@@ -0,0 +1,40 @@
+/** The function that calls the two functions below */
+function callsotherfunctions(){
+    function loadgetmovement();
+    function creategetmovementElement(movement);
+}
+
+
+/** Fetches movement hashtags from the server and adds them to the DOM. */
+function loadgetmovement() {
```

Comment:
hmmm the get movement doesnt have the same type of body as the hashtags, it is a dictionary.

Reply:
The following is what I changed, but I do not know if it is correct: 

function loadgetmovement() {
  fetch('/get-movement').then(response => response.json()).then((movements));
  const movementListElement = document.getElementById('movement-list');
  movementListElement.appendChild(getMovementElement(movement));
    
  }